### PR TITLE
[Api] Subscribed to newsletter

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

File: features/account/customer_account/subscribing_newsletter.feature [hunk 1]
```diff
@@ -6,10 +6,10 @@ Feature: Subscribing to the newsletter
 
     Background:
         Given the store operates on a single channel in "United States"
-        And there is a customer "Francis Underwood" identified by an email "francis@underwood.com" and a password "whitehouse"
```
Comment (pamil) [inline features/account/customer_account/subscribing_newsletter.feature:9]: Is this change needed here?

**Review by AdamKasp — APPROVED** [review]

File: features/account/customer_account/subscribing_newsletter.feature (hunk 1 above)
Comment (Tomanhez) [reply features/account/customer_account/subscribing_newsletter.feature:9]: Yes and not. Our behat api security service has hardcoded `sylius` password. This way was faster but I can change it 🚀

Comment (GSadee) [reply features/account/customer_account/subscribing_newsletter.feature:9]: Is the password even needed in this scenario?

Comment (Tomanhez) [reply features/account/customer_account/subscribing_newsletter.feature:9]: 👏

File: src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Customer.xml [hunk 2]
```diff
@@ -48,5 +48,11 @@
             <group>shop:customer:read</group>
             <group>shop:customer:update</group>
         </attribute>
+
+        <attribute name="subscribedToNewsletter" >
+            <group>admin:customer:read</group>
```
Comment (AdamKasp) [inline src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Customer.xml:53]: do we need this group? this scenario covers only the customer perspective.

File: src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Customer.xml [hunk 3]
```diff
@@ -48,5 +48,10 @@
             <group>shop:customer:read</group>
             <group>shop:customer:update</group>
         </attribute>
+
+        <attribute name="subscribedToNewsletter" >
```
Comment (AdamKasp) [inline src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Customer.xml:52]:
```suggestion
        <attribute name="subscribedToNewsletter">
```

**Review by AdamKasp — APPROVED** [review]

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thanks, Tomasz! :1st_place_medal:
